ChildProject/pipelines/metrics.py

```diff
@@ -1,31 +1,71 @@
 from abc import ABC, abstractmethod
+import os
 import argparse
 import datetime
 import multiprocessing as mp
 import numpy as np
 import pandas as pd
 from typing import Union, List
+import yaml
+from git import Repo
+from git.exc import InvalidGitRepositoryError
 
 import ChildProject
 from ChildProject.pipelines.pipeline import Pipeline
 
-pipelines = {}
+import ChildProject.pipelines.metricsFunctions as metfunc #used by eval
+from ..utils import TimeInterval, time_intervals_intersect
 
+pipelines = {}
 
 class Metrics(ABC):
     def __init__(
```

Comment:
I think a docstring might be useful here. I think I understood what all arguments do but I am not sure.

Reply:
👍 will add it !

If you have any doubt, the class that inherit from this class (e.g. LenaMetrics) have usually all of those arguments (and some more) . So their behavious is the same